### ISSUE TITLE
Performance cannot approach as mention

### ISSUE THREAD

**Issue by tonyli0803** [bfortuner/pytorch_tiramisu#18]
hi @bfortuner , I' m interested in your pytorch_tiramisu.

I use the train.ipynb as the backbone to train the model. However, I cannot approach the performance as you do.

My FC-DenseNet67 result converge at  0.8357994871794872.

It will be appreciate to help me figure out my questions.
Here are my questions:
  
1.    The origin code doesn't have weighted loss function. The weighted loss function do improve 
       iou, but harm to pixel accuracy. How did you get those weight?

2.    I see some past issue mention that the train.ipynb is slightly different from the original train 
       code. Do you add a lot of things on it? Can you send me the original code for further 
       comparison ?

Here is the modification from the code : 

in train.ipynb

      1. N_EPOCHS -> 1000

      2. To use multi-gpu I add this line
![image](https://user-images.githubusercontent.com/19266405/54662552-54c56900-4b19-11e9-8b40-90827dddcd08.png)

      3. To implement "exclude the 'background' class" , I add this lines. Try to set 11 class, and 
          ignore void label loss
![image](https://user-images.githubusercontent.com/19266405/54662669-b7b70000-4b19-11e9-907a-ba5f8cff7bd7.png)

       4.  I train it directly. I didn't not finetune.
![image](https://user-images.githubusercontent.com/19266405/54663012-e386b580-4b1a-11e9-96d0-e26535a2084d.png)

Please help me figure out these questions.🙏🙏🙏🙏🙇🙇🙇
My email is : tonyli0803@gmail.com
